setup.py

```diff
@@ -21,6 +21,12 @@
     'pytest>=2.8',
 ]
 
+linux_requirements = [
+    # dbus-python is a declared req only by some versions of secretstorage
```

Comment:
You can drop this line. I have moved dbus-python to `extras_require` to allow people installing it via distro packages.

Anyway `dbus-python` is also needed for the KWallet backend of python-keyring.